bin/loader-stats.js

```diff
@@ -53,8 +53,14 @@ filesToLoadPerSection = sectionsToLoad.map(section => {
 
 async function calculateSizes( section ) {
 	const fileSizePromises = section.filesToLoad.map( f => gzipSize.file( f ) );
+	let fileSizes;
 
-	const fileSizes = await Promise.all( fileSizePromises );
+	try {
+		const fileSizes = await Promise.all( fileSizePromises );
```

Comment:
You're redeclaring the `fileSizes` variable that was previously declared with `let`.

What about catching all errors that could ever be thrown from `go()`? Like doing this at the end of the script:
```js
go().catch( console.error );
```

Reply:
Even better! :)